python/lsst/dax/metaserv/metaAdminImpl.py

```diff
@@ -195,17 +197,17 @@ def addDbDescr(self, dbName, schemaFile, level, dataRel, owner,
         cmd += "VALUES('/dummy',%s,'db',%s,%s,%s,%s,%s,%s,%s) "
         opts = (str(projectId), level, dataRel, schemaVersion, dbName, schemaDescr,
                 str(ownerId), accessibility)
-        db.execCommand0(cmd, opts)
-        repoId = db.execCommand1("SELECT LAST_INSERT_ID()")[0]
+        conn.execute(cmd, opts)
+        repoId = int(db.execute("SELECT LAST_INSERT_ID()").scalar())
```

Comment:
Should we use `lastrowid` on the result of previous query instead of running `SELECT LAST_INSERT_ID()`?